go-controller/pkg/ovn/master.go

```diff
@@ -800,7 +800,8 @@ func (oc *Controller) syncNodesPeriodic() {
 		return
 	}
 
-	nodeNames := make([]string, len(nodes.Items))
+	// create a slice of a given capacity not length
```

Comment:
This comment only makes sense in the context of the diff; someone encountering it in the code later on won't understand why we're making that clarification here but not anywhere else that does the same thing. Just remove the comment.

Reply:
Yep.